vignettes/WQPDataHarmonization.Rmd

```diff
@@ -294,11 +269,16 @@ multiple HOURS to run. The total run time depends on your query inputs.
 
 ```
 
-Review all column names in the TADA Profile
+Review all column names in the TADA Profile. Remove results that are not water samples or measurements (TADA is recommended for use with water data only). 
```

Comment:
I added this. Just double checking that you don't already have this somewhere else?

Reply:
Correct. We had talked about a separate TADA function for this, but in this case, I think it makes sense to simply use the dplyr::filter() function in the vignette and/or Shiny app. It is really easy to build Shiny toggle switches and selection boxes using dplyr::filter() and similar functions; I don't think we necessarily need to create our own unless it does something else in addition to filtering and/or is used several times in the process.